src/Services/LDAPService.php

```diff
@@ -649,7 +664,7 @@ private function processThumbnailPhoto(Member $member, $fieldName, $data, $attri
         $filename = sprintf('thumbnailphoto-%s.jpg', $data['objectguid']);
         $filePath = File::join_paths($thumbnailFolder->getFilename(), $filename);
         $fileCfg = [
-            'conflict' => AssetStore::CONFLICT_OVERWRITE,
+            'conflict' => AssetStore::CONFLICT_USE_EXISTING,
             'visibility' => AssetStore::VISIBILITY_PUBLIC
```

Comment:
@ishannz  I guess the answer is here; If CONFLICT_USE_EXISTING is a check on the Filename or the contents of the file.  that should probably be a test.  e.g. Why overwrite a file if the content from LDAP is the same (one of the current problems).

Reply:
You were correct @ishannz  - it needs `CONFLICT_OVERWRITE` but we should check the hashes to make sure the contents have changed before overwriting it.